feature-libs/checkout/scheduled-replenishment/styles/components/steps/review/_review-submit.scss

```diff
@@ -0,0 +1,127 @@
+cx-schedule-replenishment-order {
```

Comment:
Please double check if we need to carry over the changes from here https://github.com/SAP/spartacus/commit/3947c79e6af547ce27b828d3903c7cc9eaa1e390#diff-fc650a9686fe1201b0347cad376e1bbf83c4f660ef4105c6ca9c5e9b34899418

Reply:
You saw changes from here because I decoupled it. This selector is inside the styles.scss folder of sheduled-replenishment

Reply:
only changes to checkout were from https://github.com/SAP/spartacus/pull/14507#discussion_r762450242
which was added to checout/base/styles